catalog/webhooks.go

```diff
@@ -0,0 +1,23 @@
+package catalog
```

Comment:
Maybe pass it to a dedicated package and avoid 'Hook' prefix everywhere?

Reply:
Will consolidate into 3 types - I don't think we need all of them - just generated them.